src/components/noticeboard/NoticeboardAlerts.tsx

```diff
@@ -0,0 +1,11 @@
+import { Alert } from 'react-native';
+
+import { texts } from '../../config';
+
+export const NoticeboardAlerts = (alertType: string) => {
```

Comment:
Generates one alert but can be used for multiple alerts. Therefore, plural naming is used.

This is only created to avoid code repetition. It can be added inside the form screens.